src/modules/config/index.ts

```diff
@@ -1,6 +1,7 @@
 import Conf from 'conf';
 
 export const defaultPlatformerAPIGateway = 'https://beta.api.platformer.com';
+export const defaultConsoleURL = 'https://beta.console.platformer.com';
```

Comment:
move these urls to a config file

Reply:
for a CLI this is not needed, we can have a seperate file called constants.ts but not a seperate config file imo